pkgdb/src/buildenv/realise.cc

```diff
@@ -63,6 +63,10 @@ namespace flox::buildenv {
 /* -------------------------------------------------------------------------- */
 
 const char * const BASH_ACTIVATE_SCRIPT = R"(
+# Disable command hashing to allow for newly installed flox packages to be found
+# immediately.
+set +h
```

Comment:
nit: we could do this after sourcing .bashrc on the off chance someone uses hash in their rc file

Reply:
Good point.